src/pages/docs/salesforce-testing/soql-as-test-step.md

```
@@ -0,0 +1,61 @@
+
+1. From the left navigation bar, navigate to **Create Tests > Test Cases**. 
+
+2. In the **Test Case Details** page, click on the test step options list and select **SOQL**. 
+
+3. Select a Salesforce connection from the **Salesforce Connection** dropdown.
+
+4. Enter your SOQL query in the editor. 
+
+For example, 
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)